app/client/src/features/events/EventDetailsCard/EventDetailsCard.tsx

```diff
@@ -32,16 +34,40 @@ interface Props {
 }
 
 export function EventDetailsCard({ fragmentRef }: Props) {
-    const { topic, title, description } = useFragment(EVENT_DETAILS_CARD_FRAGMENT, fragmentRef);
+    const { topic, title, description, startDateTime, endDateTime } = useFragment(EVENT_DETAILS_CARD_FRAGMENT, fragmentRef);
```

Comment:
Topic isn't being used here. I removed it so it would pass typechecking, but you need to either use it or remove it. I think the topic should be in this component somewhere.

Reply:
I didn't see the topic used somewhere before I made changes to the UI, and I don't see any implementation of the topic in the Figma either.